maxim/logger/together/helpers.py

```
@@ -0,0 +1,90 @@
+from collections.abc import Iterator, AsyncGenerator
+from together.types.chat_completions import ChatCompletionChunk
+from ..logger import Generation, Trace
+from ...scribe import scribe
+from typing import Optional
+from .utils import TogetherUtils
+
+class TogetherHelpers:
+    """Helper functions for Together AI SDK."""
+
+    @staticmethod
+    def sync_stream_helper(
```

Ruff (0.11.9)

Missing return type annotation for staticmethod `sync_stream_helper`

(ANN205)